src/main/java/org/apache/flink/scheduler/benchmark/JobConfiguration.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.scheduler.benchmark;
+
+import org.apache.flink.api.common.ExecutionMode;
+import org.apache.flink.runtime.io.network.partition.ResultPartitionType;
+import org.apache.flink.runtime.jobgraph.DistributionPattern;
+import org.apache.flink.runtime.jobgraph.ScheduleMode;
+
+public enum JobConfiguration {
+
+	STREAMING(DistributionPattern.ALL_TO_ALL,
+			  ResultPartitionType.PIPELINED,
+			  ScheduleMode.EAGER,
+			  ExecutionMode.PIPELINED),
+
+	BATCH(DistributionPattern.ALL_TO_ALL,
```

Comment:
Seems this enum is never used. So what config will be used for batch jobs benchmark?
 I'm also wondering whether we really need this `JobConfiguration`?
Can we simply have `createStreamingJobGraph` and `createBatchJobGraph` util methods?

Reply:
The `JobConfiguration` is added to make the name of benchmarks more readable. In this way, the name of benchmarks will be displayed as `buildTopology.STREAMING` and `buildTopology.BATCH`, rather than `buildTopologyInStreamingJob` and `buildTopologyInBatchJob`.

Reply:
Thanks for the explanation. Now I see that the enums are actually used in the `@Param` way.